sklearn/dummy.py

```diff
@@ -91,9 +92,8 @@ def fit(self, X, y, sample_weight=None):
 
         Parameters
         ----------
-        X : {array-like, sparse matrix}, shape = [n_samples, n_features]
-            Training vectors, where n_samples is the number of samples
-            and n_features is the number of features.
+        X : {array-like, sparse matrix}
+            Input vector, requires length = n_samples
```

Comment:
Hmm... This doesn't make sense. It's not ordinarily a vector but a matrix. And it can't require something that's defined due to it. I think all I mean is that `{array-like, sparse matrix}` can be extended to `{array-like, object with finite length or shape}`